app/uk/gov/hmrc/cdsreimbursementclaim/models/DeclarationInfoRequest.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.models
+
+import java.time.{Instant, ZoneId}
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+
+import play.api.libs.json.{Json, Writes}
+
+/** @param overpaymentDeclarationDisplayRequest
+  */
+final case class DeclarationInfoRequest(
+  overpaymentDeclarationDisplayRequest: OverpaymentDeclarationDisplayRequest
+)
+
+object DeclarationInfoRequest {
+
+  val dateFormat: DateTimeFormatter =
+    DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssz").withZone(ZoneId.systemDefault())
```

Comment:
Because we need to this formatter for TPI05 eventually, could you create a TimeUtils object and put this formatter there. Then it can be referenced for the work we will need to do for this and TPI05.